aesthetic_ants/penguin/tile_layer.py

```diff
@@ -63,8 +112,10 @@ def set_tile(self, tile_x: int, tile_y: int, tile_type: TileType):
                                           y=tile_y * TILE_SIZE + self.y,
                                           batch=self.batch)
 
-    def update(self, dt, **kwargs):
-        pass
+    def update(self, dt: float, **kwargs):
+        enemies = list(filter(lambda obj: isinstance(obj, Enemy),
+                              self.space.objects))
```

Comment:
I think this should be a separate method, especially since all we do with this list ultimately is make it return a number in some sort of way.

Maybe make a method called `enemy_decay_influence(self):` that for now just returns how many enemies there are?

This would also make it easier if we decide to have enemies that are even worse for the environment, like big guys, or trucks, etc.